packages/apache_tomcat/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.0"
```

```suggestion
- version: "1.5.2"
```
Since it is minor change lets only update patch version.